# Timed Auto HDR Flash
string value
---
int32 errorCode
string errorDescription